src/Control/SimpleSearchField.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace ipl\Web\Control;
+
+use ipl\Html\Attributes;
+use ipl\Html\Form;
+use ipl\Html\FormElement\InputElement;
+use ipl\Html\FormElement\SubmitElement;
+use ipl\Html\HtmlElement;
+use ipl\Web\Url;
+use ipl\Web\Widget\Icon;
+
+use function ipl\I18n\t;
+
+/**
+ * Form for simple value based search
+ */
+class SimpleSearchField extends Form
+{
+    protected $defaultAttributes = [
+        'class'     => 'search-field',
+        'name'      => 'search-field',
+        'role'      => 'search-field'
+    ];
+
+    /** @var string The term separator */
+    const TERM_SEPARATOR = ',';
+
+    /** @var string The search parameter */
+    protected $searchParameter;
+
+    /** @var Url The suggestion url */
+    protected $suggestionUrl;
+
+    /** @var string Submit label */
+    protected $submitLabel;
+
+    /**
+     * Set the search parameter
+     *
+     * @param string $name
+     *
+     * @return  $this
+     */
+    public function setSearchParameter(string $name): self
+    {
+        $this->searchParameter = $name;
+
+        return $this;
+    }
+
+    /**
+     * Get the search parameter
+     *
+     * @return string
+     */
+    public function getSearchParameter(): string
+    {
+        return $this->searchParameter ?: 'q';
+    }
+
+    /**
+     * Set the suggestion url
+     *
+     * @param   Url $url
+     *
+     * @return  $this
+     */
+    public function setSuggestionUrl(Url $url): self
+    {
+        $this->suggestionUrl = $url;
+
+        return $this;
+    }
+
+    /**
+     * Get the suggestion url
+     *
+     * @return Url
+     */
+    public function getSuggestionUrl(): Url
+    {
+        return $this->suggestionUrl;
+    }
+
+    /**
+     * Set submit label
+     *
+     * @param string $label
+     *
+     * @return  $this
+     */
+    public function setSubmitLabel(string $label): self
+    {
+        $this->submitLabel = $label;
+
+        return $this;
+    }
+
+    /**
+     * Get submit label
+     *
+     * @return string
+     */
+    public function getSubmitLabel(): string
+    {
+        return $this->submitLabel ?? t('Submit');
+    }
+
+    public function assemble()
+    {
+        $filterInput = new InputElement(null, [
+            'type'                      => 'text',
+            'placeholder'               => 'Type to search',
+            'class'                     => 'search-input',
+            'id'                        => 'search-input',
+            'autocomplete'              => 'off',
+            'required'                  => true,
+            'data-no-auto-submit'       => true,
+            'data-no-js-placeholder'    => true,
```

Comment:
What's this?

Reply:
This disables the js placeholder. Required to display the placeholder when a term has been added and the input field is empty again.